prime-router/metadata/schemas/FL/fl-covid-19.schema

```diff
@@ -335,9 +347,9 @@ elements:
 
     # column BG
     # nonstandard
-    # todo: create a value set for this based on SNOMED codes
     - name: specify_congregate_care_setting
```

Comment:
Some comment as elsewhere - I'm concerned that our code might have weird bugs with this kind of "override".  Suggest using a different name for the field, not the name in covid-19.schema.